resources/views/layouts/default.blade.php

```diff
@@ -584,95 +584,83 @@
             @endcan
 
             @can('backend.interact')
-                <li class="treeview">
-                    <a href="#">
+                <li class="treeview {!! in_array(Request::route()->getName(),App\Helpers\Helper::SettingUrls()) ? ' active': '' !!}">
+                    <a href="#" id="settings" class="">
                         <i class="fas fa-cog" aria-hidden="true"></i>
                         <span>{{ trans('general.settings') }}</span>
                         <i class="fa fa-angle-left pull-right"></i>
                     </a>
-
                     <ul class="treeview-menu">
                         @if(Gate::allows('view', App\Models\CustomField::class) || Gate::allows('view', App\Models\CustomFieldset::class))
-                            <li {!! (Request::is('fields*') ? ' class="active"' : '') !!}>
-                                <a href="{{ route('fields.index') }}">
+                            <li>
+                                <a href="{{ route('fields.index') }}" class="{!! Request::is(route('fields.index') ? ' class="active"' : '') !!}">
```

Comment:
We should leave this as `{!! (Request::is('fields*') ? ' class="active"' : '') !!}`, since field sets is also managed within this section.